<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QTabWidget" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">SettingsDialog</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="basicTab">
   <attribute name="title">
    <string>Basic</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Regional Options</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Language of the user interface:</string>
         </property>
         <property name="buddy">
          <cstring>languageComboBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SeparatorComboBox" name="languageComboBox">
         <property name="maxVisibleItems">
          <number>20</number>
         </property>
         <item>
          <property name="text">
           <string>System Default</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>User Interface</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="3">
        <widget class="QCheckBox" name="dockCheckBox">
         <property name="text">
          <string>&amp;Dock main window in system tray</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&amp;Text size in issue details and project summary:</string>
         </property>
         <property name="buddy">
          <cstring>textSizeComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="textSizeComboBox"/>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>109</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Startup Options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;Show main window at startup:</string>
         </property>
         <property name="buddy">
          <cstring>showComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="showComboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>Never</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Automatically</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Always</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&amp;Restore last connection at startup:</string>
         </property>
         <property name="buddy">
          <cstring>reconnectComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="reconnectComboBox">
         <item>
          <property name="text">
           <string>Never</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Automatically</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Always</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QCheckBox" name="autoStartCheckBox">
         <property name="text">
          <string>Start WebIssues when I start &amp;Windows</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Automatic Update</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="autoUpdateCheckBox">
         <property name="text">
          <string>&amp;Enable automatic checking for latest version of WebIssues</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="advancedTab">
   <attribute name="title">
    <string>Advanced</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Behavior</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&amp;Default action for attachments:</string>
         </property>
         <property name="buddy">
          <cstring>attachmentsComboBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="attachmentsComboBox">
         <item>
          <property name="text">
           <string>Ask what to do</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Open</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Save As...</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="title">
       <string>Update Intervals</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&amp;Update folders every:</string>
         </property>
         <property name="buddy">
          <cstring>foldersSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="foldersSpinBox">
         <property name="suffix">
          <string> min</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>60</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Update all data e&amp;very:</string>
         </property>
         <property name="buddy">
          <cstring>fullSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="fullSpinBox">
         <property name="suffix">
          <string> min</string>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="proxyGroupBox">
      <property name="title">
       <string>Proxy Settings</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="customProxyCheckBox">
         <property name="text">
          <string>Use custom pro&amp;xy settings:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>&amp;Host:</string>
           </property>
           <property name="buddy">
            <cstring>hostLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="InputLineEdit" name="hostLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>&amp;Port:</string>
           </property>
           <property name="buddy">
            <cstring>portSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="portSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>80</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>&amp;Type:</string>
           </property>
           <property name="buddy">
            <cstring>proxyTypeComboBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="proxyTypeComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>HTTP</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Socks</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Do not use a proxy for the following addresses:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="EnumLineEdit" name="excludeLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SeparatorComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/separatorcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>InputLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/inputlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>EnumLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/inputlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>languageComboBox</tabstop>
  <tabstop>dockCheckBox</tabstop>
  <tabstop>textSizeComboBox</tabstop>
  <tabstop>showComboBox</tabstop>
  <tabstop>reconnectComboBox</tabstop>
  <tabstop>autoStartCheckBox</tabstop>
  <tabstop>autoUpdateCheckBox</tabstop>
  <tabstop>attachmentsComboBox</tabstop>
  <tabstop>foldersSpinBox</tabstop>
  <tabstop>fullSpinBox</tabstop>
  <tabstop>customProxyCheckBox</tabstop>
  <tabstop>hostLineEdit</tabstop>
  <tabstop>portSpinBox</tabstop>
  <tabstop>proxyTypeComboBox</tabstop>
  <tabstop>excludeLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>dockCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>showComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customProxyCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>hostLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customProxyCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>portSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customProxyCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxyTypeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customProxyCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>excludeLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
